security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+.. _rbac_demo:
+
+Overview
```

Comment:
This doc heading should not be `Overview`...probably something like `RBAC demo...`

Reply:
Ack